packages/dropdowns/examples/multiselect.md

```diff
@@ -0,0 +1,138 @@
+The `Multiselect` component renders a customizable "tag" for each selected item.
+
+This can be any element, but the surrounding field is designed to work with
+our standard `Tag` component.
+
+```jsx static
+<Dropdown>
+  <Field>
+    <Label>Example Multiselect</Label>
+    <Multiselect
```

Comment:
This fails to dismiss the expanded menu on chevron click. Compare to https://garden.zendesk.com/react-components/autocomplete/#multiselect where clicking the chevron shows/hides the menu.

Additionally, clicking the cursor text area shows/hides the menu and it shouldn't. It's like the click action on these two controls needs to be reversed.

There's also an unfortunate rendered flash of `+ X more` onMouseDown that would be nice to have cleaned up.